content/en/sensitive_data_scanner/_index.md

```diff
@@ -0,0 +1,194 @@
+
+This document walks you through the following:
+
+- [The permissions required to view and set up scanning rules](#permissions).
+- [Setting up scanning for sensitive data](#set-up-scanning-for-sensitive-data).
+- [Investigating sensitive data issues using the Summary page](#investigate-sensitive-data-issues-in-the-summary-page).
+- [The out-of-the-box dashboard provided](#out-of-the-box-dashboard).
+
+{{< img src="sensitive_data_scanner/sds_main_12_01_24.png" alt="The Sensitive Data Scanner page showing six out of the 12 active scanning groups" style="width:90%;">}}
+
+## Permissions
+
+By default, users with the Datadog Admin role have access to view and set up scanning rules. To allow other users access, grant `data_scanner_read` or `data_scanner_write` permissions under [Compliance][10]. See [Access Control][1] for details on how to set up roles and permissions.
+
+{{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:55%;">}}
+
+
+## Set up scanning for sensitive data
+
+This section walks through the following so that you can scan your data for sensitive information:
+
+- [Adding a scanning group](#add-a-scanning-group)
+- [Adding scanning rules](#add-scanning-rules)
+- [Controlling access to events with sensitive data](#control-access-to-events-with-sensitive-data)
+- [Redacting sensitive data in tags](#redact-sensitive-data-in-tags)
+
+### Add a scanning group
+
+A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for Logs, APM, RUM, and Events. See the [Log Search Syntax][2] documentation to learn more about query filters.
+
+For Terraform, see the [Datadog sensitive data scanner group][3] resource.
+
+To set up a scanning group:
+
+1. Navigate to [Sensitive Data Scanner][4]. Alternatively, go to Organization Settings > Sensitive Data Scanner.
+1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
+1. Enter a query filter for the data you want to scan. At the top, click **APM Spans** to preview the filtered spans. Click **Logs** to see the filtered logs.
+1. Enter a name and description for the group.
+1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want.
+1. Click **Create**.
+
+### Add scanning rules
+
+A scanning rule determines what sensitive information to match within the data. Within a scanning group, add predefined scanning rules from Datadog's Scanning Rule Library or create your own rules using regex patterns. The data is scanned at the point of ingestion. For logs, the scan is done before indexing.
```

Comment:
Would like to get confirmation that this is correct:

> The data is scanned at the point of ingestion. For logs, the scan is done before indexing.

There was a request to add this information: [https://datadoghq.atlassian.net/browse/DOCS-6770](https://www.google.com/url?q=https://datadoghq.atlassian.net/browse/DOCS-6770&sa=D&source=docs&ust=1705074156216965&usg=AOvVaw1UGj6USgLYs5Bm73V0ioOW)

Reply:
Yes, it's correct, but to be even clearer we can say "The data is scanned at ingestion time during processing. For logs, this means the scan is done before indexing and other routing decisions."